src/mongocxx/test/spec/unified_tests/assert.cpp

```diff
@@ -172,6 +172,24 @@ bool is_special(T doc) {
            string::to_string(doc.get_document().value.begin()->key()).rfind("$$", 0) == 0;
 }
 
+// The C driver returns different names than the spec test. This will return the
+// name as the C driver declares it.
+bsoncxx::stdx::string_view get_alternate_name(bsoncxx::stdx::string_view key) {
```

Comment:
```suggestion
std::string get_alternate_name(const std::string& key) {
```
I retract my earlier suggestion to use `bsoncxx::stdx::string_view` after realizing that the mnmlstc implementation of `string_view` for v1.1.0 (what the CXX Driver currently depends on) is [utterly broken](https://github.com/mnmlstc/core/blob/72cccd977409e3581f05228ac027761ca4cea5dd/include/core/string.hpp#L363) (hashes using the value of the _internal pointer_ instead of the string value itself). It appears to have been fixed in [this commit](https://github.com/mnmlstc/core/commit/42a34a9b6168733d60f1334d875247d67cc364fc#diff-14a62deac1406c596d24f2f3ebdb019dab87a1caa9e35cb92e9fea83596662cb), but there is no release version containing this commit. 😐